src/Pages/EventManager/EventManager.js

```diff
@@ -50,9 +50,9 @@ export default function EventManager(props) {
   async function handleSubmit(event) {
     if (modalState === eventModalState.SUBMIT) {
       const res = await createNewEvent(event, props.user.token);
-      if(res.error === false) {
+      if (res.error === false) {
```

Comment:
is this the right way? why'd you touch this?

Reply:
autoformat